torch/_torch_docs.py

```diff
@@ -970,6 +970,64 @@ def merge_dicts(*dicts):
 Alias for :func:`torch.atanh`.
 """)
 
+add_docstr(torch.asarray,
+           r"""
+asarray(obj, *, dtype=None, device=None, copy=None, requires_grad=False) -> Tensor
+
+Creates a new :class:`Tensor` out of :attr:`obj` storage, by either copying or
+aliasing it.
```

Comment:
nit -- it might be clearer to use "view" instead of "alias"

Reply:
To be consistent with the `as_tensor` documentation (which is being updated):


"Converts :attr:`obj` into a tensor, sharing data and preserving the autograd graph if possible."

Follow-up question: is this an accurate description? See inline comment above about autograd graph preservation.

Reply:
As is, `asarray` preserves the autograd graph only if we create a view of the input tensor. I'm explicitly detaching the copied tensors. That said, I agree it's better to be consistent with `as_tensor`.